dotcom-rendering/cdk/lib/dotcom-rendering.test.ts

```diff
@@ -11,7 +11,7 @@ describe('The DotcomRendering stack', () => {
 		const app = new App();
 		const stack = new DotcomRendering(app, 'DotcomRendering', {
 			stack: 'frontend',
-			stage: 'TEST',
+			stage: 'PROD',
```

Comment:
Snapshot tests now use `PROD` stage so scaling policies are included in the snapshot